src/status_im/contexts/profile/settings/style.cljs

```diff
@@ -1,13 +1,27 @@
 (ns status-im.contexts.profile.settings.style
-  (:require [quo.foundations.colors :as colors]))
+  (:require [quo.foundations.colors :as colors]
+            [react-native.platform :as platform]))
 
 (defn navigation-wrapper
   [{:keys [customization-color inset theme]}]
   {:padding-top      inset
    :background-color (colors/resolve-color customization-color theme 40)})
 
-(def footer-container
-  {:margin-top         8
-   :margin-bottom      50
-   :padding-horizontal 20
-   :padding-vertical   12})
+(def ^:const jump-to-button-height 44)
+(def ^:const footer-padding 20)
+(def ^:const ios-bottom-offset -10)
+
+(defn footer-container
+  [bottom]
+  {:padding-horizontal footer-padding
+   :padding-top        footer-padding
+   :padding-bottom     (+ jump-to-button-height
```

Comment:
Please directly use `jump-to.constants/floating-shell-button-height`, so that we needs to change only at single place, if needed.